appveyor.yml

```diff
@@ -161,6 +162,7 @@ for:
           Exit-AppVeyorBuild
           return
       }
+      Push-AppveyorArtifact .\GitExtensionsVSIX\bin\Release\GitExtensionsVSIX.vsix
```

Comment:
Having a second thought, I don't think this is a correct way.
The artifact should be sent for signing (if signing is enabled and it is a release build), or it should be published unsigned otherwise.
All of that logic is in `Setup\Sign-Artifacts.ps1`.

The most up-to date version of `Setup\Sign-Artifacts.ps1` is in 3.0.2 branch, it may worth porting it over to the master (it will happen, a matter of timing).

Reply:
This (untested as I have not made any release builds in this branch) will upload the vsix unmodified.
Signing the vsix is an enhancement as I see it, it is not done now. Should likely be done, even if VS has not complained.
Must this be done in this PR?
(I can look into this regardless, I got privileges to SignPath project.)

Reply:
The VSIX is signed as part of the MSI installer, you can see the rules if you log into SignPath portal.

Reply:
Thinking out loud

v2.x - artifacts don't get signed
v3.0 - VSIX is included into MSI, it gets signed as part of it
v3.1+ - VSIX is separate, MSI signing must continue to work as previously. We may need a new signing policy for signing VSIX

Reply:
Signing is primarily done to ensure install time integrity. The 3.0 vsix is therefore not signed at installation time. It should be OK to distribute the vsix in the 3.0 and 3.1+ way, the official distribute is in the vs marketplace anyway. But it may be required to sign the marketplace file and it should be similar for the non-marketplace vsix as it is for non signed GE installer.

I will have to read about this. But signing the vsix should not be critical.